locales/en-US.json

```diff
@@ -156,6 +156,8 @@
     "Title": "Title",
     "Playlist privacy": "Playlist privacy",
     "Editing playlist `x`": "Editing playlist `x`",
+    "Show more": "Show more",
+    "Show less": "Show less",
```

Comment:
Please, can you add the same keys to all other translation files? When #1920 will be merged this will break CI.

```
    "Show more": "",
    "Show less": "",
```